proto/eventlog.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
```

Comment:
Can you send just this proto change in a separate PR?

(I believe the chunkstore changes were already approved in https://github.com/buildbuddy-io/buildbuddy/pull/447 -- can you submit that and then remove those changes from this PR too?)

Reply:
Sorry, I forgot to merge the other pr and rebase my branch before pushing